Gemfile.lock

```diff
@@ -319,7 +319,7 @@ DEPENDENCIES
   jbuilder (~> 2.5)
   jquery-rails (= 4.3.1)
   listen (>= 3.0.5, < 3.2)
-  pg
+  pg (~> 1.1.3)
```

Comment:
Was you intention to specify particular version with patch number or just major.minor? If first, you should specify version directly, without `~>` operator. More info [here](https://stackoverflow.com/questions/5101591/what-does-mean-in-a-gem-file)